webots_ros2_epuck/webots_ros2_epuck/driver.py

```diff
@@ -56,9 +51,16 @@
 
 DEVICE_CONFIG = {
     '@auto': True,
-    'camera': {
-        'topic_name': '',
-    }
+    'camera': {'topic_name': ''},
+    'ps0': {'always_publish': True},
+    'ps1': {'always_publish': True},
+    'ps2': {'always_publish': True},
+    'ps3': {'always_publish': True},
+    'ps4': {'always_publish': True},
+    'ps5': {'always_publish': True},
+    'ps6': {'always_publish': True},
+    'ps7': {'always_publish': True},
+    'tof': {'always_publish': True}
```

Comment:
Is it normal that the light sensors are not included here?

Reply:
They are included, but by default, they get turn on only when there are subscribers. This is a problem as we need them all the time for LaserScan. Therefore, `always_publish` will not disable the sensor when there are no subscribers.